Device/Device.cs

```diff
@@ -128,17 +97,18 @@ private async void Disconnected(MqttClientDisconnectedEventArgs e, IMqttClientOp
             {
                 Console.WriteLine("### RECONNECTING FAILED ###");
             }
+
             Console.WriteLine("Reconnected");
         }
 
-        private static string GenerateSasToken(string resourceUri, string key, int expiryInSeconds = 36000)

```

Comment:
Why remove static?

Reply:
I thought it is no longer in a static class and it is a private method whose parameters depend on properties from this class.